subsys/bluetooth/services/fast_pair/fp_crypto/fp_crypto_common.c

```diff
@@ -15,6 +16,86 @@ LOG_MODULE_REGISTER(fp_crypto, CONFIG_FP_CRYPTO_LOG_LEVEL);
 #include "fp_crypto.h"
 #include "fp_common.h"
 
+#define ADDITIONAL_DATA_SHA_POS		0
+#define ADDITIONAL_DATA_SHA_LEN		8
+#define ADDITIONAL_DATA_NONCE_POS	(ADDITIONAL_DATA_SHA_POS + ADDITIONAL_DATA_SHA_LEN)
+#define ADDITIONAL_DATA_ENC_DATA_POS	(ADDITIONAL_DATA_NONCE_POS + FP_CRYPTO_PN_NONCE_LEN)
+
```

Comment:
Could you please use 2 empty lines between data and first function implementation?